test/study-webidl.js

```diff
@@ -77,10 +77,18 @@ interface WhereIAm {};
 
   it('reports no anomaly for valid EventHandler attributes definitions', async () => {
```

Comment:
There is already a generic "no anomaly when IDL is valid" test. This one was more directly intended to target valid IDL with some event handler definition.

Most of the tests are written to make sure that anomalies get reported. It will likely be useful to have tests as well that make sure that anomalies are not incorrectly reported. We could perhaps merge these tests in one with a large Web IDL blob. I'll do that later on.

Reply:
oh, I see, I misread the IDL; ignore me then :)